tests/jerry/regexp-simple-atom-and-iterations.js

```diff
@@ -56,3 +56,15 @@ assert (r.exec ("abcdefghijkl") == "abcdefghijkl");
 
 r = /\n/;
 assert (r.exec ("\n") == "\n");
+
+assert (/[\12]+/.exec ("1\n\n\n\n\n2") == "\n\n\n\n\n");
+assert (/[\1284]+/.exec ("1\n\n8\n4\n\n2") == "\n\n8\n4\n\n");
+assert (/[\89]12/.exec ("1\9128123") == "912");
+assert (/[\11]/.exec ("1\n\n\t\n\n2") == "\t");
+assert (/[\142][\143][\144]/.exec ("abcde") == "bcd");
+
+assert (/\12+/.exec ("1\n\n\n\n\n2") == "\n\n\n\n\n");
+assert (/\11/.exec ("1\n\n\t\n\n2") == "\t");
+assert (/\142\143\144/.exec ("abcde") == "bcd");
+assert (/\942\143\144/.exec ("a942cde") == "942cd");
+assert (/\14234/.exec ("b34") == "b34");
```

Comment:
Please add test case which contains capturing brackets and backreferences fall back (or not) to octal parsing.